include/hip/hcc_detail/hip_runtime_api.h

```diff
@@ -1359,6 +1358,7 @@ hipError_t hipMalloc3DArray(hipArray_t *array,
  *  @see hipMemcpy, hipMemcpyToArray, hipMemcpy2DToArray, hipMemcpyFromArray, hipMemcpyToSymbol, hipMemcpyAsync
  */
 hipError_t hipMemcpy2D(void* dst, size_t dpitch, const void* src, size_t spitch, size_t width, size_t height, hipMemcpyKind kind);
+hipError_t hipMemcpy_2D(const hip_Memcpy2D* pCopy);
```

Comment:
What is the reason for both (subtly different) APIs ?

Reply:
hipMemcpy2D is equivalent to cudaMemcpy2D  (CUDA Runtime API) whereas 
hipMemcpy_2D is equivalent to cuMemcpy2D (CUDA Driver API).

Reply:
This is pretty subtle.  Can we come up with a syntax that is more consistent with the rest of HIP - perhaps hipMemcpyParam2D to indicate this version uses copy parms inside a structure rather than as arguments?

Reply:
Agree, we should use better name. Initially I used hipDrvMemcpy2D though wanted to avoid using "Drv" only at few places, so discarded that.